src/pages/LogIn.jsx

```diff
@@ -1,65 +1,147 @@
-import {useRef, useState, useEffect} from 'react';
+import { useRef, useState, useEffect } from 'react';
+import axiosInstance from '../api/axios';
 
 function LogIn() {
   const userRef = useRef();
   const errRef = useRef();
 
-  const [user, setUser] = useState('');
-  const [pwd, setPwd] = useState('');
+  const EMAIL_REGEX = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
+  const PWD_REGEX = /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#$%]).{8,24}$/;
```

Comment:
For Login, don't need to do a password regex (it should just comeback as invalid if it doesn't match from the backend)

Reply:
Got you. Removed